2022q4/pot.adoc

```diff
@@ -0,0 +1,23 @@
+=== Containers and FreeBSD: Pot, Potluck and Potman
+
+Links: +
+link:https://github.com/bsdpot[Pot organization on github] URL: link:https://github.com/bsdpot[https://github.com/bsdpot]
+
+Contact: Luca Pizzamiglio (Pot) <pizzamig@freebsd.org> +
+Contact: Stephan Lichtenauer (Potluck) <sl@honeyguide.eu> +
+Contact: Michael Gmelin (Potman) <grembo@freebsd.org>
+
+Pot is a jail management tool that link:https://www.freebsd.org/news/status/report-2020-01-2020-03/#pot-and-the-nomad-pot-driver[also supports orchestration through Nomad].
+
+During the last quarter, link:https://github.com/bsdpot/pot/releases/tag/0.15.4[pot 0.15.4] was released.
+It again contains a number of improvements like signing pot images as well as many bug fixes. Also, we welcome two new pot contributors link:https://github.com/zilti[@zilti] and link:https://github.com/reezer[@reezer].
+
+Additionally, there ist a new link:https://galaxy.ansible.com/zilti/pot[Ansible pot collection] available.
+
+Potluck aims to be to FreeBSD and pot what Dockerhub is to Linux and Docker: a repository of pot flavours and complete container images for usage with pot and in many cases Nomad.
+
+As you can see, we link:https://github.com/bsdpot/potluck/commits/master[had a busy quarter again], this time including improvements to the link:https://github.com/bsdpot/potluck/tree/master/nextcloud-nginx-nomad[Nextcloud] as well as link:https://github.com/bsdpot/potluck/tree/master/jitsi-meet[Jitsi] images.
+
+Also, Luca's EuroBSDCon 2022 talk link:https://www.youtube.com/watch?v=b0IB0mc2KTE[is now available on YouTube].
```

Comment:
For accessibility to screenreaders,

> Also, link:https://www.youtube.com/watch?v=b0IB0mc2KTE[Luca's EuroBSDCon 2022 talk] is now available on YouTube.

Doing it this way means that if reading out a list of links separately from the main body of text (a common method), the link text won't be "is now available on YouTube" with no obvious way to figure out what is.

Reply:
Fixed.